src/main/java/com/sessions/session6/DateAndTimeHomework.java

```diff
@@ -0,0 +1,108 @@
+package com.sessions.session6;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.Period;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.Timer;
+
+public class DateAndTimeHomework {
+    public static void main(String[] args) {
+
+        //1. Display today`s date
+
+//        LocalDate todayDate = LocalDate.now();
+//        System.out.println(todayDate);
+
+
+        //2. Write a method that prints a LocalDate with the following format 12-2005-20 (month-year-day)
+
+//        LocalDate localDate = LocalDate.now();
+//        String formattedLocalDate = localDate.format(DateTimeFormatter.ofPattern("MM-yyyy-dd"));
+//        System.out.println(formattedLocalDate);
+
+
+        //3. Checking if today is 2019-12-22
+
+//        LocalDate todayDate = LocalDate.now();
+//        LocalDate checkDate = LocalDate.of(2019, 12, 22);
+//        boolean datesCompared = todayDate.equals(checkDate);
+//        System.out.println("Is today 2019-12-22?\n" + datesCompared);
+
+
+        //4. Judging whether two dates read from console are equal
+
+//        Scanner scanner = new Scanner(System.in);
+//        DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+//        boolean validInput1 = true;
```

Comment:
Ai putea pune o conditie de iesire din loop (de exemplu sa verifici daca inputul nu este egal cu "quit" sau "exit" ). 
Momentan validInput1 de exemplu e setat pe true tot timpul, deci mereu intra in bucla (daca nu se arunca o exceptie sau eroare pe undeva).